Src/ILGPU.Algorithms/InitializeExtensions.cs

```diff
@@ -64,9 +64,8 @@ internal static void InitializeKernel<T>(Index1 index, ArrayView<T> view, T valu
         {
             var result = accelerator.LoadAutoGroupedKernel(
                 (Action<Index1, ArrayView<T>, T>)InitializeKernel,
-                out int groupSize,
-                out int minGridSize);
-            minDataSize = groupSize * minGridSize;
+                out var info);
+            minDataSize = info.MinGroupSize.Value * info.MinGridSize.Value;
```

Comment:
ISSUE: Calling `Nullable<>.Value` will throw an exception if the nullalble is not populated.

Could maybe use `Nullable<>.GetValueOrDefault()` or change `minDataSize` to `int?`

Reply:
This would be an option. An `autogrouped` kernel loader should always provide this information (assuming that it has been implemented properly). That's why I simply converted these nullable values to "raw" values. However, I guess we should provide a fallback in this case to avoid issues in the future.